dist/munin/obs_srcserver_packages

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+case $1 in
+  config)
+       cat <<'EOM'
+graph_title OBS Source Server: The number of packages
+graph_vlabel Source packages
+graph_category obs
+graph_order numpackages
+numpackages.label obs_src_server_number_of_packages
+numpackages.info The number of source packages in obs source server
+numpackages.type GAUGE
+EOM
+       exit 0;;
+esac
+
+num_numpackages=`ls /srv/obs/sources/ | wc -l`
```

Comment:
It will not make a much IO because we use ls command to know the all files in same folder (dentry in filesystem). 

$ time ls /srv/obs/sources/ | wc -l 
1490

real	0m0.009s
user	0m0.009s
sys	0m0.002s